.github/workflows/build.yaml

```diff
@@ -0,0 +1,46 @@
+name:
+
+on:
+  push:
+    branches:
+    - '*'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [ 1.13.x ]
```

Comment:
Can we use 1.15 like the Dockerfile has?

Reply:
fixed